### PR TITLE
HRIS-218 [FE] Integrate re-assign employee schedule functionality

### PR DESCRIPTION
**Pull request by JohnRoyCabezas**
## Issue Link

[Backlog Link](https://framgiaph.backlog.com/view/HRIS-218)

## Definition of Done

- [x] Users can re-assign other members to another schedule

## Pre-condition

run docker
go to `http://localhost:3000/schedule-management` and re-assign a member as shown in the video below.

## Expected Output

Users can re-assign other members to another schedule

## Screenshots/Recordings

Proof:

[screen-capture (1).webm](https://user-images.githubusercontent.com/109579325/235092743-9161f513-18ec-4333-8ead-d6a6874d9134.webm)

### CODE REVIEW

File: client/src/components/molecules/ViewScheduleMembersModal/ReassignMemberScheduleModal.tsx [hunk 1]
```diff
@@ -53,14 +61,19 @@ const ReassignMemberScheduleModal: FC<Props> = ({ isOpen, closeModal, member }):
     }
   })
 
-  const handleSave: SubmitHandler<ReassigneScheduleFormValues> = async (data): Promise<void> => {
+  const handleSave: SubmitHandler<any> = async (data): Promise<void> => {
```
Comment (jsvelte) [inline client/src/components/molecules/ViewScheduleMembersModal/ReassignMemberScheduleModal.tsx:64]: Why you put any in the types? Adjust your code if you have type error

**Review by jsvelte — CHANGES_REQUESTED** [review]
Please resolve the type error and don't practice to used `any` types

File: client/src/components/molecules/ViewScheduleMembersModal/ReassignMemberScheduleModal.tsx (hunk 1 above)
Comment (JohnRoyCabezas) [reply client/src/components/molecules/ViewScheduleMembersModal/ReassignMemberScheduleModal.tsx]: Forgot to change it back. Updated! Thanks.

**Review by jsvelte — APPROVED** [review]
# Approved 💯

**Review by NiloJr-sun — APPROVED** [review]

**Review by arden-dave — APPROVED** [review]
Approve. 👍

**Review by abduljalilpalala — APPROVED** [review]
### Approved! ✅

**Pull request merged**